opds.py

```diff
@@ -1001,7 +982,7 @@ def add_breadcrumbs(self, lane, annotator, include_lane=False):
             )
             
             # Add links for all visible ancestors that aren't root
-            for ancestor in reversed(lane.visible_ancestors()):
+            for ancestor in lane.parentage:
```

Comment:
I think this still needs to be reversed - the order of the breadcrumbs is important.

Reply:
I took it out because I thought `parentage` returned the parents in the right order but you're right, it doesn't.